src/admin/data/index.ts

```diff
@@ -0,0 +1,45 @@
+import { grpc } from "@improbable-eng/grpc-web"
+import {
+  GCStagedRequest,
+  GCStagedResponse,
+  PinnedCidsRequest,
+  PinnedCidsResponse,
+} from "@textile/grpc-powergate-client/dist/powergate/admin/v1/admin_pb"
+import { AdminServiceClient } from "@textile/grpc-powergate-client/dist/powergate/admin/v1/admin_pb_service"
+import { Config } from "../../types"
+import { promise } from "../../util"
+
+export interface Data {
+  /**
+   * Unpins staged data not related to queued or executing jobs.
+   * @returns An object containing a list of unpinned cids.
+   */
+  gcStaged: () => Promise<GCStagedResponse.AsObject>
+
+  /**
+   * Get pinned cids information of hot-storage.
+   * @returns Pinned cids information of hot-storage.
+   */
+  pinnedCids: () => Promise<PinnedCidsResponse.AsObject>
+}
```

Comment:
These methods were added in a previous powergate update but never added to the js client.